### DISCUSSION TITLE
How important is it to have X chromosome probes included in the methylation data?

### DISCUSSION THREAD

**Discussion by SamuliSalminen** [genetics-of-dna-methylation-consortium/godmc_phase2#19] [General]
According to 
[Normalised methylation data page](https://github.com/genetics-of-dna-methylation-consortium/godmc_phase2/wiki/Normalised-methylation-data):

"_You should keep nonspecific binding probes, probes with SNPs in their sequence, multimapping probes. We will filter out these probes at the end._"

During the last week's meeting I asked that if it is OK for us to use methylation data that have these probes already filtered from it. If I remember correctly, Josine answered that it is OK as long as the data include X chromosome probes.

It turns out that also the X chromosome probes are already filtered from our methylation data. So, the question is that **how crucial it is that the methylation data include the X chromosome probes?** 
(It would be a very large and hard task for us to reconstruct the methylation data so that the X chromosome probes would be also included).

**Comment by epzjlm** [comment]
As we are running separate analyses on chrX for a chrX project we would like you to keep chromosome X probes. However if these probes are removed and it is a lot of effort to put them back in we think you should go ahead without them.
